test/spec/algoliasearch.helper/distinct.js

```diff
@@ -1,69 +1,66 @@
 'use strict';
 
-var test = require('tape');
 var forEach = require('lodash/forEach');
 
 var algoliasearchHelper = require('../../../index.js');
 var requestBuilder = require('../../../src/requestBuilder');
 
 var fakeClient = {};
 
-test('Distinct not set', function(t) {
+test('Distinct not set', function() {
   var helper = algoliasearchHelper(fakeClient, null, {
     facets: ['facetConj'],
     disjunctiveFacets: ['facet']
   });
   var state0 = helper.state;
 
   var disjunctiveFacetSearchParam = requestBuilder._getDisjunctiveFacetSearchParams(helper.state);
-  t.equal(disjunctiveFacetSearchParam.distinct, undefined, '[disjunctive] distinct should be undefined');
+  expect(disjunctiveFacetSearchParam.distinct).toBe(undefined);
```

Comment:
We can use `toBeUndefined()`.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453